### DISCUSSION TITLE
Get current song info with global shortcut

### DISCUSSION THREAD

**Discussion by alienator** [strawberrymusicplayer/strawberry#1325] [Q&A] [answered]
Hi, it is  possible to get the current song info, using a global shortcut?

Lets say I am working with gimp, and listening an online radio, I like the song currently played, so I hit the global shortcut to get that info, and pasted into a text file, all this without having to change from gimp to strawberry.

**Comment by saintamh** [comment, marked as answer]
I use `playerctl` for that (on Ubuntu 22.04)

* `playerctl --player=strawberry metadata` prints out all metadata for the current track
* `playerctl --player=strawberry metadata xesam:title` specifically specifies the title

On Linux you could also try using `notify-send` to display the title as desktop notification, perhaps something like this:

```bash
notify-send "$(playerctl --player=strawberry metadata xesam:title)"
```

**Comment by alienator** [comment]
Thanks that is useful
